decidim-initiatives/config/locales/en.yml

```diff
@@ -203,6 +203,10 @@ en:
         content_blocks:
           highlighted_initiatives:
             max_results: Maximum amount of elements to show
+            order:
+              default: Default
```

Comment:
What's this "default" order? As a user I'd be confused 😕

Reply:
The default order is the unordered one.